onyo/cli/cat.py

```diff
@@ -14,7 +14,9 @@
     'asset': dict(
         metavar='ASSET',
         nargs='+',
-        help='Paths of assets to print'
+        help=r"""
+            Paths of assets to print
```

Comment:
I know this change is just string-moving here, but comparing with the lines in config.py/edit.py there should be a trailing dot here too.